nexus/src/external_api/params.rs

```diff
@@ -107,6 +107,7 @@ pub struct InstanceMigrate {
 pub struct VpcCreate {
     #[serde(flatten)]
     pub identity: IdentityMetadataCreateParams,
+    pub ipv6_prefix: Option<Ipv6Net>,
```

Comment:
Is `Ipv6Net` the correct type to use here? I'd guess any `IPv6` prefix is a proper `IPv6Net` interface but the inverse isn't true. If we expressly created an `Ipv6Prefix` type we could give more expressive results in the API to inform its usage. That'd also move the conversion logic out of the VPC model and isolate it to this type.

This isn't blocking.

Reply:
Thanks for taking a look @zephraph! If I understand your question, you're wondering if we might allow specifying _only_ the 48-bit prefix that unique local addresses share? Or even the 40 random bits, as the first 8 are required to be `fd`.

If so, that'd probably be more confusing than helpful, I'd expect. Networking folks would likely expect to work with the full, standard subnet mask notation, and the type wouldn't really be used anywhere but this specific API call.